### PR TITLE
LIME-82: Add Connections page

### LINKED ISSUE #82: Add Connections page to the Profile Settings
- [ ] We should use our design (screenshot just example)
- [ ] Add option to connect Strava
- [ ] Add option to connect Google Fit

Disconnected state:
Image

Connected state:
Image

### PR DESCRIPTION
**Pull request by ArtemLymarenko**
Added connections page.

Result.
p-1
p-2

**600 pixels**
p-3
p-4

### CODE REVIEW

File: frontend/src/bundles/profile/pages/connections-page/components/connection-option.tsx [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+                                </span>
+                            )}
+                        </p>
+                    </div>
+                    <Button
+                        variant={isConnected ? 'secondary' : 'primary'}
+                        size={'small'}
+                        label={isConnected ? 'Disconnect' : 'Connect'}
+                        className={
+                            'h-6 max-w-[180px] sm:h-10 sm:max-w-40 md:max-w-60'
```
Comment (SofiiaTrokhymchuk) [inline frontend/src/bundles/profile/pages/connections-page/components/connection-option.tsx]: it's better to use rem not px

File: frontend/src/bundles/profile/pages/connections-page/components/connection-option.tsx [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+};
+
+const ConnectionOption = ({
+    title,
+    description,
+    iconName,
+}: Properties): JSX.Element => {
+    const [isConnected, setIsConnected] = useState(false);
+
+    const handleClick = useCallback((): undefined => {
```
Comment (JKaypa) [inline frontend/src/bundles/profile/pages/connections-page/components/connection-option.tsx]: Instead of undefined, I think you can better place void

**Comment by nikita-remeslov** [comment]
Do you use this red color?
image

**Review by nikita-remeslov — CHANGES_REQUESTED** [review]
Lets center icon and button for mobile view
image

File: frontend/src/bundles/profile/pages/connections-page/components/styles.module.css [hunk 3]
```diff
@@ -0,0 +1,9 @@
+@media (width >= 600px) {
```
Comment (nikita-remeslov) [inline frontend/src/bundles/profile/pages/connections-page/components/styles.module.css:1]: we should use our screen sizes. check tailwind config

Comment (nikita-remeslov) [reply frontend/src/bundles/profile/pages/connections-page/components/styles.module.css:1]: also you can add classes like `lg:flex md:block....`

**Review by nikita-remeslov — CHANGES_REQUESTED** [review]

**Comment by ArtemLymarenko** [comment]
Updated layout

File: frontend/src/bundles/common/components/icon/types/icon.type.ts [hunk 4]
```diff
@@ -1,3 +1,8 @@
-type IconName = 'arrowDown' | 'logoIcon' | 'notFoundIcon';
+type IconName =
```
Comment (nikita-remeslov) [inline frontend/src/bundles/common/components/icon/types/icon.type.ts]: lets create enum for it

**Review by dimapopovych — APPROVED** [review]

**Review by DavidCharif — APPROVED** [review]

**Review by valery-chumak — APPROVED** [review]

**Review by stpalamar — APPROVED** [review]

**Review by nikita-remeslov — APPROVED** [review]

**Pull request merged**